app/models/general_content.rb

```diff
@@ -3,6 +3,7 @@ class GeneralContent < Node
 
   validates_with SectionHeritageValidator
   validates_presence_of :section
+  validates :parent, presence: true
```

Comment:
Isn't this redundant? As we are testing ancestry depth. Or if you want to keep it for readability purposes, you could combine the two lines as: 
`validates :parent, presence: true, ancestry_depth: { minimum: 2 }`